tests/server/server-executeQuery-005.phpt

```diff
@@ -50,8 +50,10 @@ printf("Set profile level to 0 successfully: %s\n", (empty($result->ok) ? 'no' :
 --EXPECTF--
 Set profile level to 2 successfully: yes
 object(stdClass)#%d (%d) {
-  ["$query"]=>
-  object(stdClass)#%d (%d) {
+  ["find"]=>
+  string(%d) "%s"
+  ["filter"]=>
+  object(stdClass)#%d (1) {
```

Comment:
This looks like something that needs to be split into two tests.
One on pre 3.2, and one 3.2+